lib/sonar/cli/cli.rb

```diff
@@ -27,7 +29,12 @@ def usage
 
     desc 'search_all [QUERY TERM]', 'Search all Sonar record types'
     def search_all(term)
-      Sonar::Search::QUERY_TYPES_MAP.keys.each do |type|
+      if term =~ IS_IP
+        types = ip_search_type_names
+      else
+        types = domain_search_type_names
+      end
+      types.each do |type|
         search(type, term) rescue 'Search failed'
```

Comment:
IMO, any result with an empty `collection` should be hidden.  Any errors should be shown, but that one in particular should go away with #18